pages/whyd/terminal.vue

```diff
@@ -38,5 +39,76 @@ export default {
+  animation: grain 1.5s steps(6) infinite;
```

Comment:
I'm imagining that we don't need the grain to be animated. But if we do want that, then maybe we could hook in the tiniest bit of JS+`requestAnimationFrame` to control it. LGTM, then I can help you hook up some dynamic noise later